File: week7/count_word.py
#!/usr/bin/python

import sys
import fileinput
import re


words = [];
target = sys.argv[1];
for lines in sys.stdin:
	my_regex = re.compile(r'\b%s\b' % target, re.IGNORECASE);
	words += re.findall(my_regex, lines);
print target.lower() + " occurred " + repr(len(words)) + " times";

File: week7/count_word.pl
#!/usr/bin/perl -w
use strict;

my $count = 0;
my $target = $ARGV[0];
while (<STDIN>) {
	#$words += scalar(split(/[a-zA-Z]/, $_));
	my @items = split(/[^A-Za-z]/, $_);
	my @new;
	foreach(@items){
	    if( ( defined $_) and ($_ =~ /^$target$/i)){
	        $count++;
	    }
	}
}
my $word = lc($target);
print "$word occurred $count times";